modules/ROOT/pages/dw-core-functions-groupby.adoc

```diff
@@ -4,12 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 
 
+
 [[groupby1]]
-== groupBy(Array<T>, (item: T, index: Number) -> R): { &#40;R&#41;: Array<T> }
+== groupBy&#40;Array<T&#62;, &#40;item: T, index: Number&#41; &#45;&#62; R&#41;: { &#40;R&#41;: Array<T&#62; }
 
 Classifies the elements of a list (array) using the specified `criteria` function.
 
-This function returns an object containing the grouping criteria as keys.
+The resulting object will have the grouping criteria as keys.
```

Comment:
You don't have to change, but I don't think we need "will have" here, "contains" is an active concrete verb.